en/extras.xliff

```diff
@@ -0,0 +1,411 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+  <file source-language="en" datatype="plaintext" original="file.ext">
+    <body>
+      <trans-unit id="currencies.AUD">
+        <source>A$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.BRL">
+        <source>R$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.CAD">
+        <source>CA$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.CNY">
+        <source>CN¥</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.EUR">
+        <source>€</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.GBP">
+        <source>£</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.HKD">
+        <source>HK$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.ILS">
+        <source>₪</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.INR">
+        <source>₹</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.JPY">
+        <source>¥</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.KRW">
+        <source>₩</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.MXN">
+        <source>MX$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.NZD">
+        <source>NZ$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.PHP">
+        <source>₱</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.TWD">
+        <source>NT$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.USD">
+        <source>$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.VND">
+        <source>₫</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.XAF">
+        <source>FCFA</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.XCD">
+        <source>EC$</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.XOF">
+        <source>F CFA</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.XPF">
+        <source>CFPF</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="currencies.XXX">
+        <source>¤</source>
+        <target></target>
+    </trans-unit>
+      <trans-unit id="languages.bg">
```

Comment:
To avoid starting with a lot of missing translations, we can probably populate most of these from Firefox?
https://hg.mozilla.org/l10n/gecko-strings/file/default/toolkit/toolkit/intl/languageNames.ftl

Tranvision provides an API to get all translation, e.g. for Bulgarian (bg -> language-name-bg)
https://transvision.flod.org/string/?entity=toolkit/toolkit/intl/languageNames.ftl:language-name-bg&repo=gecko_strings